components/Pagination/Pagination.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { number, string } from 'prop-types';
+import classNames from 'classnames';
+import LeftAngleIcon from 'static/images/icons/FontAwesome/angle-left-solid.svg';
+import RightAngleIcon from 'static/images/icons/FontAwesome/angle-right-solid.svg';
+import PaginationItem from './PaginationItem/PaginationItem';
+import styles from './Pagination.css';
+
+Pagination.propTypes = {
+  className: string,
+  currentPage: number.isRequired,
+  totalPages: number.isRequired,
+};
+
+Pagination.defaultProps = {
+  className: undefined,
+};
+
+// eslint-disable-next-line react/prop-types
+const PaginationItems = ({ currentPage, totalPages }) => {
```

Comment:
I would love a dev-only error that highlights when current page is > total pages (impossible)

Reply:
Yup, also I guess it would be a good idea to check if `currentPage` is less than or equal to `0`.